rules/__tests__/no_large_snapshots.test.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+const noLargeSnapshots = require('../no_large_snapshots');
+
+// was not able to use https://eslint.org/docs/developer-guide/nodejs-api#ruletester for these because there is no way to configure RuleTester to run non .js files
```

Comment:
Is there an issue for that?

If not, should we create one?

Reply:
I'm looking more into this. I know it wasn't working properly for me when I tried using `ruleTester` to test this rule and I thought I had read in their docs that it does not support non `.js` extensions but I can't find where I read that anymore.

Reply:
News on this?